src/averageAges.js

```diff
@@ -20,6 +20,17 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const sortedPeople = century
+    ? people.filter((person) =>
+      person.sex === 'm' && Math.ceil(person.died / 100) === century
```

Comment:
Consider using constants with meaningful names instead of magic strings and numbers it's a considered anti-pattern